### DISCUSSION TITLE
Releasing 19

### DISCUSSION THREAD

**Discussion by johanvos** [gluonhq/scenebuilder#581] [General]
Hi,

There are 2 PRs merged, and I think we can now release Scene Builder 19, which will be using JavaFX 19 and JDK 19. 
We can move the pending PRs and issues for the next milestone (SB20)
@AlmasB and @Oliver-Loeffler Are you ok with that?

**Comment by AlmasB** [comment]
All ok

**Comment by Oliver-Loeffler** [comment]
Okay for me.

**Comment by Oliver-Loeffler** [comment]
Hello all!
With some luck we can get some issues solved and possibly the one or other PR merged within next 2 months.
What about publishing then Scene Builder 19.1.0 as soon as some more bug fixes / functionality has been added? Even maintenance / refactorings could be published.

What do you think about that?

**Reply by Oliver-Loeffler** [reply]
Ah, I see - there is a better thread for this question #551.

**Comment by Oliver-Loeffler** [comment]
SB 19 has been released, hence this can be closed.
